workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginInstallContent.tsx

```diff
@@ -98,8 +331,16 @@ export const MarketplacePluginInstallContent = ({
         >
           Copy
         </Button>
-      </div>
-    </div>
+        <Button
+          variant="outlined"
+          color="primary"
+          sx={{ ml: 2 }}
+          onClick={() => navigate(-1)}
```

Comment:
This doesn't work if you just open that page. Can we please use a ButtonLink instead with a calculated like this in the Install Header.

See PluginInstallHeader:

```tsx
const PluginInstallHeader = () => {
  const params = useRouteRefParams(pluginInstallRouteRef);
  const plugin = usePlugin(params.namespace, params.name);

  const displayName = plugin.data?.metadata?.title ?? params.name;
  const title = `Install ${displayName}`;
  const pluginLink = useRouteRef(pluginRouteRef)({
    namespace: params.namespace,
    name: params.name,
  });

  return <Header title={title} type="Plugin" typeLink={pluginLink} />;
};
```

Reply:
How about going back to the `extensions/plugins` page? I think this is a more natural behavior, and we can confirm with Shiran about this.

```suggestion
          onClick={() => navigate('/extensions/plugins')}
```

Also, if a user goes to the install page from any pages such as home page, I don't think going back to the original page is a bad idea, logically. Let's confirm with Shiran on this.